test/stored_queries/test_djornl.py

```diff
@@ -0,0 +1,227 @@
+"""
+Tests for the Dan Jacobson ORNL Arabidopsis stored queries.
+"""
+import json
+import time
+import unittest
+import requests
+import os
+import glob
+import yaml
+
+from test.helpers import get_config, assert_subset, modified_environ
+from test.stored_queries.helpers import create_test_docs
+from importers.djornl.parser import DJORNL_Parser
+
+_CONF = get_config()
+_NOW = int(time.time() * 1000)
+_TEST_DIR = '/app/test'
+_VERBOSE = 0
+
+
+def print_db_update(response, collection):
+    if not _VERBOSE:
+        return
+    print(f"Saved docs to collection {collection}!")
+    print(response.text)
+    print('=' * 80)
+
+
+class Test_DJORNL_Stored_Queries(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+
+        # import the results file
+        results_file = os.path.join(_TEST_DIR, 'djornl', 'results.json')
+        with open(results_file) as fh:
+            lines = fh.read()
+
+        cls.json_data = json.loads(lines)
```

Comment:
The results for all the stored queries were computed by my own fair hands and stored in this file.